doc/script_commands.txt

```diff
@@ -11114,3 +11114,24 @@ See 'achievementinfo' for valid <type> values.
 - Excludes ACHIEVEINFO_LEVEL and ACHIEVEINFO_SCORE.
 
 ---------------------------------------
+
+*addfame(<amount>,{,<char id>})
+
+Increases the fame of the attached player or the supplied <char id> by the <amount> given.
+Note: Only works with Blacksmith, Alchemist and Taekwon classes.
```

Comment:
```suggestion
Note: Only works with classes that use the ranking system.
```
Would be good to generalize it, so that we do not have to think about updating all references, when a rank is added to a class.